Source/common/SNTFileInfo.mm

```diff
@@ -165,7 +164,7 @@ - (instancetype)initWithPath:(NSString *)path {
 - (void)hashSHA1:(NSString **)sha1 SHA256:(NSString **)sha256 {
   const int MAX_CHUNK_SIZE = 256 * 1024;  // 256 KB
   const size_t chunkSize = _fileSize > MAX_CHUNK_SIZE ? MAX_CHUNK_SIZE : _fileSize;
-  char *chunk = malloc(chunkSize);
+  char *chunk = (char *)malloc(chunkSize);
```

Comment:
Now that this is an .mm file, use c++ style castings.
https://google.github.io/styleguide/cppguide.html#Casting